backend/tests/controllers/test_results/test_test_results.py

```diff
@@ -635,162 +738,6 @@ def test_ordering_consistency(
             assert len(our_results) == 2
 
 
-class TestGetEnvironments:
```

Comment:
I don't believe these belong here? And they look to be duplicated by the tests that do exist where I'd expect them to be.